Makefile

```diff
@@ -67,6 +67,17 @@ coverage: ## check code coverage quickly with the default Python
 	coverage html
 	$(BROWSER) htmlcov/index.html
 
+vcr: # remake VCR cassettes and run other integration tests
+	cci org scratch qa pytest
+	cci org scratch_delete pytest
+	find . -name \Test*.yaml | xargs rm
+	pytest --org qa --run-slow-tests -rs --replace-vcrs
+
+slow_tests: vcr # remake VCR cassettes and run other integration tests
+	cci org scratch_delete pytest
+	pytest integration_tests/ --org pytest -rs
```

Comment:
This won't work if the `pytest` org hasn't been defined yet. Maybe it should be predefined in CumulusCI's cumulusci.yml instead of via `cci org scratch`?

Reply:
fe16f83